lib/install.js

```diff
@@ -142,6 +143,15 @@ exports.install = function(targets, options) {
       return install(name, targets[name], opts);
     }))
     .then(function() {
+      var hooks = config.pjson.hooks;
+      if(hooks) {
+        if('postinstall' in hooks && typeof hooks.postinstall === 'string') {
+          System.config(config.getLoaderConfig());
+          return System.import(hooks.postinstall);
```

Comment:
The only problem with a direct import approach is that it won't work so well for hooks that need to delay further operations as we only get a synchronous execution context (until top-level await becomes a thing that is!).

If this is fine for you I'm happy to go with this, but it could be worth thinking about.